### PR TITLE
playground.final

### PR DESCRIPTION
**Pull request by kayachakmak**

### CODE REVIEW

File: css/css-responsive_playground/css/styles.css [hunk 1]
```diff
@@ -62,7 +62,7 @@ h1 {
 .box--one {
   background-color: lightblue;
 
-  width: 100px;
```
Comment (ghost) [inline css/css-responsive_playground/css/styles.css:65]: I'm not sure if we were supposted to cut the size in the half. I believe the % is right tough.

File: css/css-responsive_playground/css/styles.css [hunk 2]
```diff
@@ -73,24 +73,48 @@ h1 {
...
 .box--four {
   background-color: lightgray;
 
   width: 100px;
   height: 100px;
 }
 
+@media (prefers-color-scheme: dark) {
+  .box--four {
+    background-color: var(--color-nemo);
```
Comment (ghost) [inline css/css-responsive_playground/css/styles.css:111]: Oh, I just chose white. Nice!

**Review by ghost — COMMENTED** [review]
Pretty much the same changes I made. Good job.

File: css/css-responsive_playground/css/styles.css (hunk 1 above)
Comment (kayachakmak) [reply css/css-responsive_playground/css/styles.css:65]: You are definitely correct!

**Pull request merged**